tests/test_http.py

```diff
@@ -18,25 +19,23 @@
 
 class TestHTTPRequests(TestCase):
 
-    @classmethod
-    def setUpClass(cls):
-        cls.s = Session("not an email", "not a password", site_url="https://app.crunch.io/api/")
-        cls.r = cls.s.get("http://httpbin.org/headers")
```

Comment:
We shouldn't rely on httpbin.org being online and replying with the answer we need.
I got multiple failures both locally and on github actions because the site was not online